benchmark/simple/main.js

```diff
@@ -57,7 +57,7 @@ function main() {
     let absNetworkFile;
     if(typeof networkFile === 'undefined') {
         try{
-            absNetworkFile = Util.resolvePath('network/fabric-v1.1/2org1peergoleveldb/fabric-go-tls.json');
+            absNetworkFile = Util.resolvePath('network/fabric-v1.1/2org1peergoleveldb/fabric-node-tls.json');
```

Comment:
Any reason for changing the default? Plus PR #287 will make the parameters mandatory, so there won't be any default values anyway.